spec/features/admin/variants_spec.rb

```diff
@@ -58,6 +58,47 @@
     v.unit_description.should == 'bar'
   end
 
+  describe "editing on hand and on demand values", js:true do
+    let(:product) { create(:simple_product) }
+    let(:variant) { product.variants.first }
+
+    before do
+      login_to_admin_section
+    end
+
+    it "allows changing the on_hand value" do
+      visit spree.edit_admin_product_variant_path(product, variant)
+
+      page.should have_field "variant_on_hand", with: variant.on_hand
+      page.should have_unchecked_field "variant_on_demand"
+
+      fill_in "variant_on_hand", with: "123"
+      click_button 'Update'
+      page.should have_content %Q(Variant "#{product.name}" has been successfully updated!)
```

Comment:
I think these tests miss `page.should have_field "variant_on_hand", with: "123"` here.

Reply:
yes, I agree...